src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ContestedOrderControllerTest.java

```diff
@@ -121,6 +127,40 @@ public void finalOrderSuccessWithoutAnyFinalOrder() throws Exception {
             .andExpect(jsonPath(path + DOCUMENT_BINARY_URL, is(BINARY_URL)));
     }
 
+    @Test
+    public void sendOrderSuccessWhenOfflineNotificationsEnabled() throws Exception {
+        doCaseDataSetUpWithoutAnyHearingOrder();
+        whenStampingDocument().thenReturn(caseDocument());
+        when(featureToggleService.isOfflineNotificationsEnabled()).thenReturn(true);
+
+        ResultActions result = mvc.perform(post(SEND_ORDER_ENDPOINT)
```

Comment:
I know the method you're calling already exists but it would make our lives easier when we come to refactor this if you cann the method directly with an object rather than over HTTP with JSON

Reply:
this can still be addressed

Reply:
I will raise a tech debt ticket and work on this refactoring work over the christmas period

Reply:
https://tools.hmcts.net/jira/browse/FR-1937 raised